discord/channel.py

```diff
@@ -1231,6 +1246,13 @@ def __init__(self, *, me, state, data):
         self.me = me
         self.id = int(data['id'])
 
+    async def __contains__(self, item):
```

Comment:
```suggestion
    def __contains__(self, item):
```
Not sure why this is coroutine?

Reply:
Just my miss, fixed